.sync.yml

```diff
@@ -35,6 +35,9 @@ Rakefile:
 
 appveyor.yml:
   matrix_extras:
-    -
```

Comment:
Why is this deleted?

Reply:
oops!

Reply:
the
```
RUBY_VERSION: 24-x64
CHECK: spec_pester`
```

is still deleted, and it's not in appveyor.